assets/src/components/detours/diversionPage.tsx

```diff
@@ -410,6 +410,7 @@ export const DiversionPage = ({
   return (
     <>
       <article
+        data-fs-element="Detours"
```

Comment:
This is the only one that gives me pause, as it is generic, but perhaps that's exactly what we want! Whenever a detour modal is opened, no user actions taken, the "Detours" event occurs

Reply:
This is really for scoping. If we ever had a "Undo" button elsewhere in the application, the `data-fs-element="Undo"` selector that FS uses would trigger for both of those. I _think_ it's _better_ to use more generic names and scope them via this type of "context" selector, since fullstory allows you to compose them